frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -545,6 +545,17 @@ class Worksheet extends React.Component {
         });
     };
 
+    updateSchemaItem = (rows, start_index, original_length) => {
+        // rows: list of string representing the new schema:
+        //      % schema example
+        //      % add uuid uuid [0:8]
+        // start_index: the index in source corresponding to % schema example
+        // original_length: how many fields the original schema occupied, one for the example (starting from add)
```

Comment:
Not clear what original_length is. Would it be equal to 0 in the example above?

Reply:
Updated